src/test/scala/cromwell/binding/values/WdlValueSpec.scala

```diff
@@ -0,0 +1,101 @@
+package cromwell.binding.values
+
+import java.nio.file.Paths
+
+import cromwell.binding.{WdlExpression, WdlNamespace}
+import cromwell.util.SampleWdl
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.{FlatSpec, Matchers}
+
+class WdlValueSpec extends FlatSpec with Matchers {
+
+  import TableDrivenPropertyChecks._
+
+  behavior of "WdlValue"
+
+  val wdlValueRawStrings = Table(
+    ("wdlValue", "rawString"),
+    (WdlBoolean(false), "false"),
+    (WdlBoolean(true), "true"),
+    (new WdlFile(Paths.get("hello/world/path")), "hello/world/path"),
```

Comment:
Whoops, didn't notice it was a `case class` and needed the `Path` constructor. Thanks!